packages/lesswrong/components/tagging/TagSubforumPage.tsx

```diff
@@ -68,11 +69,16 @@ const styles = (theme: ThemeType): JssStyles => ({
     },
   },
   aside: {
-    width: 380,
+    maxWidth: 380,
+    flexBasis: 0,
+    flexGrow: 1,
```

Comment:
This makes it so the central column takes up its full width (765px) when the asides are visible (and doesn't change width depending what content is in it)